### PR TITLE
Uitpas auth links

### PR DESCRIPTION
**Pull request by jurgen-publiq**
Heb een beetje gesukkeld met de commit en push. Is in principe enkel overal vervangen van authenticatie methode door URL naar die pagina.

### CODE REVIEW

**Comment by bertramakers** [comment]
@jurgen-publiq should this be reviewed by anyone in particular? I don't think anyone was aware this was open.

**Comment by jurgen-publiq** [comment]
Well... I have no idea who should review it. It were a bunch of inaccuracies in the documentation for the UiTPAS API.
Basically check if I didn't make mistakes.

**Comment by bertramakers** [comment]
@brusselsregular @corneelwille Maybe one of you has more knowledge about this repo than I do to review this?

File: content/contributions/latest/propose-fixes/through-github.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+---
+# Propose a fix using the gituhb page
```
Comment (bertramakers) [inline content/contributions/latest/propose-fixes/through-github.md:3]:
```suggestion
# Propose a fix using the github page
```

File: content/contributions/latest/propose-content.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+---
+# Propose content
```
Comment (bertramakers) [inline content/contributions/latest/propose-content.md:3]: @jurgen-publiq can you add some info what the purpose of these new pages is? They seem unfinished and have no actual info right now.

Comment (bertramakers) [reply content/contributions/latest/propose-content.md:3]: Maybe something better suited for a separate PR?

**Comment by bertramakers** [comment]
I also noticed this partially overlaps with https://github.com/cultuurnet/uit-documentatie/pull/19/files

**Pull request closed without merging**